packages/web/src/components/apps/statusbar.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import { BugIcon, ChevronDownIcon, ChevronRightIcon, TrashIcon } from 'lucide-react';
+import { useHotkeys } from 'react-hotkeys-hook';
+
+import { Button } from '@srcbook/components/src/components/ui/button';
+import { cn } from '@/lib/utils.ts';
+import { LogMessage, useLogs } from './use-logs';
+
+function getLabelForError(error: LogMessage) {
+  switch (error.type) {
+    case 'vite_error':
+      return 'Error running vite preview server!';
+    case 'npm_install_error':
+      return 'Error running npm install!';
+  }
+}
+
+type CollapsibleErrorMessageProps = {
+  error: LogMessage;
+};
+
+function CollapsibleErrorMessage({ error }: CollapsibleErrorMessageProps) {
+  const [open, setOpen] = useState(false);
+  return (
+    <>
+      <button
+        className={cn(
+          'flex items-center gap-2 font-mono p-2 first:border-t first:border-t-border focus-visible:outline-none ring-inset focus-visible:ring-1 focus-visible:ring-ring',
```

Comment:
Just checking that we need `first:border-t-border`. I think that color is automatic, unless maybe there's an override that I'm missing, you can just remove